rules/filename-case.js

```diff
@@ -133,31 +133,19 @@ function englishishJoinWords(words) {
 		return words[0];
 	}
 
-	if (words.length === 2) {
-		return `${words[0]} or ${words[1]}`;
-	}
-
-	return `${words.slice(0, -1).join(', ')}, or ${words[words.length - 1]}`;
+	return words.length === 2 ? `${words[0]} or ${words[1]}` : `${words.slice(0, -1).join(', ')}, or ${words[words.length - 1]}`;
```

Comment:
This is fine, although maybe on 3 lines
```suggestion
	return words.length === 2 ?
		`${words[0]} or ${words[1]}` :
		`${words.slice(0, -1).join(', ')}, or ${words[words.length - 1]}`;
```

Reply:
But the old one still better

```js
function englishishJoinWords(words) {
	if (words.length === 1) {
		return words[0];
	}

	if (words.length === 2) {
		return `${words[0]} or ${words[1]}`;
	}

	return `${words.slice(0, -1).join(', ')}, or ${words[words.length - 1]}`;
}
```

Reply:
Indeed

Reply:
Problem is hard to detect this. because maybe some other stuff above `words.length === 2`